### ISSUE TITLE
Play the video only after play button is clicked 

### ISSUE THREAD

**Issue by GET100PERCENT** [libre-tube/LibreTube#2198]
### Describe your suggested feature

Normally, libretube plays videos just after any video is clicked.

If the default video resolution is large, then it will use a lot of data, which is not data friendly (on metered networks).

So there can be an option "to choose whether to play the videos automatically when the video page is opened or play the video only after the play button is clicked" in the settings.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by GET100PERCENT** [comment]
For both the functions.

**Comment by GET100PERCENT** [comment]
Newpipe has already implemented it.

**Comment by limonovthesecond2** [comment]
How this can be implemented with the existing "Autoplay" option: Queues are created only after playback has started. Therefore, the existing "Autoplay" will only work with queues, regardless of this option.

**Comment by GET100PERCENT** [comment]
There will be two options Autoplay and Auto eneque next stream just like new pipe.
![Screenshot_20221209-180455_NewPipe SponsorBlock](https://user-images.githubusercontent.com/116144024/206704604-6ab70a02-cc8f-4e77-b983-f7c1ab83962c.jpg)
![Screenshot_20221209-180500_NewPipe SponsorBlock](https://user-images.githubusercontent.com/116144024/206704677-3367223c-197b-42ca-bee8-4dda7b377245.jpg)

**Comment by akano12** [comment]
> Normally, libretube plays videos just after any video is clicked.
>

Not aware of different reasons on why people click on videos other than to play them

> If the default video resolution is large, then it will use a lot of data, which is not data friendly (on metered networks).
> 
>So there can be an option "to choose whether to play the videos automatically when the video page is opened or play the video only after the play button is clicked" in the settings.
>
How about no?
You already have the option to pick a default resolution when steaming on  mobile data and wifi.
Adding this option is literally useless

**Comment by GET100PERCENT** [comment]
I think it will be very useful.
I aware of the default resolution option of libretube.
People do not watch all the full videos on which they have clicked .
If the video is not useful, they close it.
Upto 6 videos in every 10 videos are not useful.

In that case changing the default resolution each time is very bothering.

**Comment by GET100PERCENT** [comment]
For example if you want to watch a video about "reproduction of angiosperms" , you will get thousands of videos in YouTube, then in order to know which video is best for your purpose, you have to go through the comments of a number of videos.

In that case the Autoplaying of videos will result in a much data loss.

Changing the default resolution each time is very bothering.

**Comment by limonovthesecond2** [comment]
> > If the default video resolution is large, then it will use a lot of data, which is not data friendly (on metered networks).
> > So there can be an option "to choose whether to play the videos automatically when the video page is opened or play the video only after the play button is clicked" in the settings.
> 
> How about no? You already have the option to pick a default resolution when steaming on mobile data and wifi. Adding this option is literally useless

1. A user doesn't always want to watch a video immediately after opening it. He may want to just read the description, comments, add the video to a queue and only then start playback.
2. Using mobile network, it would be awesome to save bits without automatic playing videos. After reading the description, the user may not want to play the video or play it in "Audio only" mode to save bits.
3. This feature is required to use this software on poor networks where only the "Audio only" mode works stably.

**Comment by limonovthesecond2** [comment]
> There will be two options Autoplay and Auto eneque next stream just like new pipe. ![Screenshot_20221209-180455_NewPipe SponsorBlock](https://user-images.githubusercontent.com/116144024/206704604-6ab70a02-cc8f-4e77-b983-f7c1ab83962c.jpg) ![Screenshot_20221209-180500_NewPipe SponsorBlock](https://user-images.githubusercontent.com/116144024/206704677-3367223c-197b-42ca-bee8-4dda7b377245.jpg)

"Auto-enqueue next stream" is available in LibreTube at Player => Queue => Insert related videos.
You are talking about autoplay in queues that cannot be changed in NewPipe. LibreTube currently has an "Autoplay" option for queues, which would be better renamed to "Autoplay for queue" for example. Therefore, create a new "Autoplay" option to play a video when you open it.

**Comment by GET100PERCENT** [comment]
> > There will be two options Autoplay and Auto eneque next stream just like new pipe.
> 
> "Auto-enqueue next stream" is available in LibreTube at Player => Queue => Insert related videos. You are talking about autoplay in queues that cannot be changed in NewPipe. LibreTube currently has an "Autoplay" option for queues, which would be better renamed to "Autoplay for queue" for example. Therefore, create a new "Autoplay" option to play a video when you open it.

Yes, exactly the same I want.

**Comment by Bnyro** [comment]
Videos now only play automatically when the data saver mode is disabled, a separate option isn't worth adding.

**Comment by GET100PERCENT** [comment]
Okay 👍

**Comment by ThomasLeigh** [comment]
However, it would be of a great help to be able to completely disable video autoplay within the app even regardless of data saving mode.

**Comment by ThomasLeigh** [comment]
Besides, one of the main reasons I've switched from NewPipe to LibreTube is it's beautiful subscription page with large video covers. Many times what I need to do is just to download a video - in such a case I always need to pause it first because I don't want it to be played. If, however, I enable data saving mode those beautiful covers disappear and using the app is not so attractive any more :( .
